src/components/accessories/admission/PatientAdmission.tsx

```diff
@@ -86,18 +102,58 @@ const PatientAdmission: FC = () => {
     scrollToElement(null);
   };
 
+  const fields: TFields<AdmissionFormFieldName> = {
+    ...initialFields,
+    ward: {
+      value: currentAdmission?.ward?.code ?? "",
+      type: "text",
+    },
+    admType: {
+      value: currentAdmission?.admType?.code ?? "",
+      type: "text",
+    },
+    diseaseIn: {
+      value: currentAdmission?.diseaseIn?.code?.toString() ?? "",
+      type: "text",
+    },
+    admDate: {
+      value: currentAdmission?.admDate ?? "",
+      type: "date",
+    },
+    note: {
+      value: currentAdmission?.note ?? "",
+      type: "text",
+    },
+    bedDays: {
+      value: differenceInDays(
+        new Date(+(currentAdmission?.admDate ?? "")),
+        new Date()
+      ).toString(),
+      type: "number",
+    },
+    transUnit: {
+      value: currentAdmission?.transUnit?.toString() ?? "",
+      type: "text",
+    },
+  };
```

Comment:
I think that we could move this inside a const.ts file

Reply:
or config

Reply:
May be a config. We cannot use a const since we have to use the currentAdmission which is dynamic data.

Reply:
Yeah you right, config sounds great